community/STEERING-COMMITTEE.md

```diff
@@ -1,61 +1,118 @@
+The Knative Steering Committee meets bi-weekly or as-needed.
+Given the nature of the discussions in the committee, meetings are not generally
+open the public. Questions around governance are listed as issues in the
+[docs repo](https://github.com/knative/docs), and we invite your feedback there.
 
-  - Approve members of the Tech Oversight Committee.
+## Committee members
 
-- Management of project assets
+Seats on the Steering Committee are held by a company, not by the individual.
 
-  - Control access to, establish processes regarding, and provide a final
-    escalation path for any Knative repository.
+The current membership of the Steering committee is (listed alphabetically by name):
 
-  - Guided by the TOC for normal business.
+| &nbsp;                                                   | Member         | Company | Profile                                  |
+| -------------------------------------------------------- | -------------- | ------- | ---------------------------------------- |
+| <img width="30px" src="https://github.com/dewitt.png">   | DeWitt Clinton | Google  | [@dewitt](https://github.com/dewitt)     |
+| <img width="30px" src="https://github.com/mchmarny.png"> | Mark Chmarny   | Google  | [@mchmarny](https://github.com/mchmarny) |
+| <img width="30px" src="https://github.com/isdal.png">    | Tomas Isdal    | Google  | [@isdal](https://github.com/isdal)       |
 
-  - Control and delegate access to and establish processes regarding other
-    project resources/assets not covered by the above, including web sites and
-    their domains, blogs, social-media accounts, etc.
+There are 4 additional seats to fill at this time.
 
-  - Manage the Knative brand to decide which things can be called “Knative” and
-    how that mark can be used in relation to other efforts or vendors.
+### Allocation of seats
 
-## Committee Mechanics
+Seats on the steering committee are allocated based upon contribution
+to the project. No final decision has been made on the exact formula.
+As the project continues to grow, we expect to add additional seats to the committee,
+ensuring that those contributing to the project are properly represented.
 
-The committee's operation and charter may be changed by unanimous consent of
-committee members.
+At a minimum, seat allocation is reviewed once per year.
 
-In case of extended absence, an absent member will be considered to be in
-agreement after 8 days have passed since a proposal was accepted by all present
-members. An absent member may appoint a single delegate during absences.
+Members of the committee may step down at any time. When a member steps down, the
+company who holds the seat shall nominate a new individual for the seat.
 
-<!-- TODO ## Committee Meeting -->
+Changes to the number of seats, which company seats are allocated to, and nominations
+to the committee are confirmed by majority vote of the committee members.
 
-## Committee Members
+## Frequently asked questions
```

Comment:
I would not include this, or I would rework it. The title "Frequently asked questions" suggests a non-binding, informative section. But it's included in the same document as rules and appears to include statements of binding rules (eg about amendments, consensus by lack of objection etc).

Reply:
Thanks -- I think there are important details in here however -- I've thought about renaming this as "Clarifications", or should I try to fold these details back into the above points?

Reply:
I think it's worth pushing them back up into the main body of the document. Expressing rules in two different forms makes it harder to understand the document.

It may be that you start from a Q&A format to help you elucidate scenarios you want to cover, but I would not render the binding document that way.